code/graphics/opengl/gropengldeferred.cpp

```diff
@@ -260,16 +266,18 @@ void gr_opengl_deferred_lighting_finish()
 			case Light_Type::Cone:
 			case Light_Type::Point:
 				gr_bind_uniform_buffer(uniform_block_type::Lights, buffer.getAlignerElementOffset(element_index),
-				                       sizeof(graphics::deferred_light_data), buffer.bufferHandle());
+					sizeof(graphics::deferred_light_data), buffer.bufferHandle());
+
 				gr_opengl_draw_deferred_light_sphere(&l.vec);
-				++element_index;
+				++element_index; 
 				break;
 			case Light_Type::Tube:
 				gr_bind_uniform_buffer(uniform_block_type::Lights, buffer.getAlignerElementOffset(element_index),
-				                       sizeof(graphics::deferred_light_data), buffer.bufferHandle());
+					sizeof(graphics::deferred_light_data), buffer.bufferHandle());
 
 				vec3d dir, newPos;
 				matrix orient;
+
```

Comment:
You appear to have a bunch of whitespace-only changes in here that ought to be cleaned up